src/components/Avatar.jsx

```diff
@@ -0,0 +1,16 @@
+import '../styles/Avatar.css'
+
+function Avatar({ userProfile }) { 
+  const { email, profileImageSource } = userProfile
```

Comment:
UI를 그리는데 필요한 데이터만 받도록 prop을 구성해도 좋겠습니다!
또 'User'라는 도메인을 꼭 알지 않아도 될 것 같아요!
상황에 따라 user가 꼭 한가지 모델에 종속적이지 않을 수도 있습니다.
예를 들어 역할과 권한에 따라 'user', 'manager'와 같이 사용자의 종류가 나뉜다던지, bot 같은 가상 화자에 대한 avatar가 필요해 질 수도 있겠죠?

```js
```suggestion
function Avatar({ text, imageSource }) { 
```
```

이렇게 추상적으로 필요한 데이터만 받도록 하기를 권해드립니다!